patches/chrome-browser-renderer_context_menu-render_view_context_menu.cc.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+index 78628f4cca9eb1aa6dce06daf0d6cc0b11e212f4..9b34f346d5e112f6f1569dce9fb3070cfa7d2e9a 100644
+--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
++++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+@@ -1055,6 +1055,10 @@ void RenderViewContextMenu::AppendLinkItems() {
+           IDC_CONTENT_CONTEXT_OPENLINKOFFTHERECORD,
+           is_app ? IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD_INAPP
+                  : IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
++      menu_model_.AddItemWithStringId(
++          IDC_CONTENT_CONTEXT_OPENLINKTOR,
++          is_app ? IDS_CONTENT_CONTEXT_OPENLINKTOR_INAPP
++                 : IDS_CONTENT_CONTEXT_OPENLINKTOR);
+ 
+     } else {
+       menu_model_.AddItemWithStringId(IDC_CONTENT_CONTEXT_OPENLINKNEWTAB,
+@@ -1067,6 +1071,8 @@ void RenderViewContextMenu::AppendLinkItems() {
+ 
+       menu_model_.AddItemWithStringId(IDC_CONTENT_CONTEXT_OPENLINKOFFTHERECORD,
+                                       IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
++      menu_model_.AddItemWithStringId(IDC_CONTENT_CONTEXT_OPENLINKTOR,
++                                      IDS_CONTENT_CONTEXT_OPENLINKTOR);
+     }
+ 
+     AppendOpenWithLinkItems();
+@@ -1779,6 +1785,9 @@ bool RenderViewContextMenu::IsCommandIdEnabled(int id) const {
+     case IDC_CONTENT_CONTEXT_OPENLINKOFFTHERECORD:
+       return IsOpenLinkOTREnabled();
+ 
++    case IDC_CONTENT_CONTEXT_OPENLINKTOR:
```

Comment:
Can we subclass RenderViewContextMenu and override IsCommandIdEnabled?
In overridden IsCommandIdEnabled check case of `IDC_CONTENT_CONTEXT_OPENLINKTOR` first and return `RenderViewContextMenu::IsCommandIdEnabled`

Reply:
0e99684fd8fa0e122a2ce95881f03e606dba64e9

Reply:
uhh, hold on, that change is not right